lib/Client.js

```diff
@@ -96,7 +96,7 @@ class Client extends EventEmitter {
     * @arg {Number} [options.firstShardID=0] The ID of the first shard to run for this client
     * @arg {Boolean} [options.getAllUsers=false] Get all the users in every guild. Ready time will be severely delayed
     * @arg {Number} [options.guildCreateTimeout=2000] How long in milliseconds to wait for a GUILD_CREATE before "ready" is fired. Increase this value if you notice missing guilds
-    * @arg {Number | Array<String>} [options.intents] A list of intents, or raw bitmask value describing the intents to subscribe to. Some intents, like `guildPresences` and `guildMembers`, must be enabled on your application's page to be used. By default, all non-privileged intents are enabled.
+    * @arg {Number | Array<String>} [options.intents] A list of intent strings or numbers, or a raw bitmask value describing the intents to subscribe to. Some intents, like `guildPresences` and `guildMembers`, must be enabled on your application's page to be used. By default, all non-privileged intents are enabled.
```

Comment:
```suggestion
    * @arg {Number | Array<String>} [options.intents] A list of [intent names](https://abal.moe/Eris/docs/reference) or numbers, or a raw bitmask value describing the intents to subscribe to. Some intents, like `guildPresences` and `guildMembers`, must be enabled on your application's page to be used. By default, all non-privileged intents are enabled.
```

I'm trying to think of a better way to specify that you provide the actual shifted result but can't think of any, but linking to the actual reference for intent names would be useful